.github/workflows/c-api-from-buffer.yaml

```diff
@@ -120,3 +120,99 @@ jobs:
           ./streaming-zipformer-buffered-tokens-hotwords-c-api
 
           rm -rf sherpa-onnx-streaming-zipformer-*
+
+      - name: Test streaming paraformer with tokens and hotwords loaded from buffers
+        shell: bash
+        run: |
+          gcc -o streaming-paraformer-buffered-tokens-hotwords-c-api ./c-api-examples/streaming-paraformer-buffered-tokens-hotwords-c-api.c \
+            -I ./build/install/include \
+            -L ./build/install/lib/ \
+            -l sherpa-onnx-c-api \
+            -l onnxruntime
+
+          ls -lh streaming-paraformer-buffered-tokens-hotwords-c-api
+
+          if [[ ${{ matrix.os }} == ubuntu-latest ]]; then
+            ldd ./streaming-paraformer-buffered-tokens-hotwords-c-api
+            echo "----"
+            readelf -d ./streaming-paraformer-buffered-tokens-hotwords-c-api
+          fi
+
+          curl -SL -O https://github.com/k2-fsa/sherpa-onnx/releases/download/asr-models/sherpa-onnx-streaming-paraformer-bilingual-zh-en.tar.bz2
+          tar xvf sherpa-onnx-streaming-paraformer-bilingual-zh-en.tar.bz2
+          rm sherpa-onnx-streaming-paraformer-bilingual-zh-en.tar.bz2
+
+          ls -lh sherpa-onnx-streaming-paraformer-bilingual-zh-en
+          echo "---"
+          ls -lh sherpa-onnx-streaming-paraformer-bilingual-zh-en/test_wavs
+
+          export LD_LIBRARY_PATH=$PWD/build/install/lib:$LD_LIBRARY_PATH
+          export DYLD_LIBRARY_PATH=$PWD/build/install/lib:$DYLD_LIBRARY_PATH
+
+          ./streaming-paraformer-buffered-tokens-hotwords-c-api
```

Comment:
By the way, we don't support hotwords when paraformer is used.

Only transducer models can support hotwords.

Reply:
thx for the noting, I didnot use hotwords for the paraformer, this is a copy-issue